sites/meo.pt/meo.pt.channels.xml

```diff
@@ -203,7 +203,7 @@
   <channel site="meo.pt" lang="pt" xmltv_id="TVCineEmotion.pt" site_id="TVCEMOH">TVCine Emotion</channel>
   <channel site="meo.pt" lang="pt" xmltv_id="TVCineTop.pt" site_id="TVCTOPH">TVCine TOP</channel>
   <channel site="meo.pt" lang="pt" xmltv_id="TVI.pt" site_id="TVI">TVI</channel>
-  <channel site="meo.pt" lang="pt" xmltv_id="TVIFiccao.pt" site_id="TVIFIC">TVI Ficção</channel>
+  <channel site="meo.pt" lang="pt" xmltv_id="VmaisTVI.pt" site_id="VMAISTVI">V+ TVI</channel>

```

Comment:
Invalid xmltv_id.

https://github.com/iptv-org/epg/blob/master/CONTRIBUTING.md#how-to-add-a-channel-to-the-guide

Reply:
It literally isnt invalid. The channel changed name and programming. You should accept it.